relaton/serializers/bibxml/reference.py

```diff
@@ -120,7 +120,13 @@ def create_reference(item: BibliographicItem) -> Element:
     except ValueError:
         pass
     else:
-        ref.set('target', target)
+        if item.doctype == 'internet-draft':
+            ref.append(E.format(
+                type="TXT",
+                target=target
```

Comment:
For style consistency, in this project we put a trailing comma after each argument.